travis/update-docker-hub.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eo pipefail
+
+docker build -t mitodl/bootcamp_web_travis -f Dockerfile .
```

Comment:
Just curious what the 'mitodl/bootcamp_web_travis' docker repo is for, and when it would make sense to run this script?